src/streamlink/plugins/ltv_lsm_lv.py

```diff
@@ -1,45 +1,128 @@
 import logging
 import re
-from urllib.parse import urlparse
+from urllib.parse import urlsplit, urlunsplit
 
 from streamlink.plugin import Plugin, pluginmatcher
-from streamlink.plugin.api.utils import itertags
-from streamlink.stream.hls import HLSStream
+from streamlink.plugin.api import validate
+from streamlink.stream.hls import HLSStream, HLSStreamReader, HLSStreamWorker
 
 log = logging.getLogger(__name__)
 
 
+def copy_query_url(to, from_):
+    """
+    Replace the query string in one URL with the query string from another URL
+    """
+    return urlunsplit(urlsplit(to)._replace(query=urlsplit(from_).query))
+
+
+class LTVHLSStreamWorker(HLSStreamWorker):
+    def process_sequences(self, playlist, sequences):
+        super(LTVHLSStreamWorker, self).process_sequences(playlist, sequences)
```

Comment:
```suggestion
        super().process_sequences(playlist, sequences)
```
Let's not go backwards in time... Every other super-call in the code base is in the implicit py3 style and we should keep it like that.
```
$ GIT_PAGER=cat git grep -E 'super\\([^\\)]'
src/streamlink/plugins/ltv_lsm_lv.py:        super(LTVHLSStreamWorker, self).process_sequences(playlist, sequences)
```

Reply:
> Let's not go backwards in time...

Reply:
Overzealous IDE compatibility auto-correct.